qmp/socket.go

```diff
@@ -68,6 +68,26 @@ func NewSocketMonitor(network, addr string, timeout time.Duration) (*SocketMonit
 	return mon, nil
 }
 
+// NewListeningSocketMonitor listens for a monitor socket connection from Qemu.
+// An error is returned if the connection fails to accept.
+func NewListeningSocketMonitor(network, addr string) (*SocketMonitor, error) {
```

Comment:
I'd suggest just calling this function `Listen`.

Reply:
Updated to 'Listen'.